airflow/utils/python_virtualenv_script.jinja2

```diff
@@ -20,6 +20,18 @@
 import {{ pickling_library }}
 import sys
 
+# Check whether Airflow is available in the environment.
+# If it is, we'll want to ensure that we integrate any macros that are being provided
+# by plugins prior to unpickling the task context.
+if int(sys.version[0]) >= 3:
```

Comment:
Airflow 2.0 only supports Python 3.6 and up. https://github.com/apache/airflow#requirements

Reply:
I'll make the check a bit more strict, just in case :-)